browser-extension/common/src/popup/components/common/RuleItem/index.tsx

```diff
@@ -42,10 +45,15 @@ const RuleItem: React.FC<RuleItemProps> = ({
     };
 
     handleUpdateRule(updatedRule);
+    sendEvent(EVENT.RULE_TOGGLED, {
+      tab,
```

Comment:
Let's drop `tab`. It's too low-level detail to capture. Also, it is increasing coupling in code (child being aware of which parent it is using it).